.github/workflows/integration-tests.yml

```diff
@@ -311,6 +315,91 @@ jobs:
           else
             echo "run_command=./smoke/${{ matrix.product.name }}_test.go" >> "$GITHUB_OUTPUT"
           fi
+      - name: Check for "enable tracing" label
+        id: check-label
+        run: |
+          label=$(jq -r '.pull_request.labels[].name' "$GITHUB_EVENT_PATH")
+          if [[ "$label" == "enable tracing" ]]; then
+            echo "Enable tracing label found."
+            echo "trace=true" >> $GITHUB_OUTPUT
+          else
+            echo "Enable tracing label not found."
+            echo "trace=false" >> $GITHUB_OUTPUT
+          fi
+          echo "$PWD"
+      - name: Setup Grafana and OpenTelemetry
+        id: docker-setup
+        if: steps.check-label.outputs.trace == 'true' && matrix.product.name == 'ocr2' && matrix.image.tag-suffix == '-plugins'
+        run: |
+          # Create network
```

Comment:
Would it make sense to instead build this out in testcontainers-go like we are the rest of the containers? Not sure if this is something we are going to want in other repositories like chainlink-solana, if we would like it there then having it be more reusabe would make sense.

Reply:
Hmm, I hadn't thought before about implementing this in testcontainers-go. My general apprehension is creating coupling between infra and code, but since we ship changes to workflows in the same repo as core, I think that apprehension no longer applies. 

It depends a bit on the cohesion of the infra spun up with testcontainers-go. If everything that is needed to be configured lives in this repo, then I think it's a potential improvement. If it requires tuning across multiple repos, I would be cautious to make that change.

> then having it be more reusabe would make sense
This workflow is portable - there's no coupling to the state of core that I'm aware of. The only issue with bringing this workflow to other repos is resource conflicts over ports on the remote server instance, but we can work around that.